### ISSUE TITLE
`Terminal: navigate accessible buffer` does not work sometimes

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#177697]
found by @jooyoungseo

**Comment by meganrogge** [comment]
I can sometimes repro this on mac - looks like occasionally, the cursor will get reset to the bottom instead of going to the new position

**Comment by meganrogge** [comment]
Repro was to create a narrow split terminal, run a few commands with decent output

**Comment by eleanorjboyd** [comment]
here are the steps I took- not sure what I am missing here to repro the exact behavior
1. turn on voice over on mac
2. enter split terminal
3. turn on screen reader optimized
4. turn on navigate accessible buffer
5. tried the command ⇧Tab to enter the buffer 
6. now it cycled through a few different options including "editor not accessible at this time", "toolbar", "terminal access toolbar" but then I get stuck on the editor and can't go back to the terminal using the same key- it also seems that I cant move to the other terminal in split terminal if that was the intent.

Sorry for such a long explanation!

**Comment by meganrogge** [comment]
- turn on voice over on mac
- enter split terminal
- turn on screen reader optimized

the above are all correct. then:
- focus a terminal, adjust the width via dragging so it's pretty narrow, and run a few commands
- shift+tab so you can now see the accessible buffer
- ctrl/cmd shift+o (windows/mac)  so you see a quick-pick with commands to go to 
- select some commands to navigate to
